scripts/run-spark.py

```diff
@@ -136,7 +152,7 @@ def run_output(cmd, **kwargs):
 
 
 def generate_input(args):
-    run("hdfs dfs -rm -r -f /terasort/input")
+    run("hdfs dfs -rm -r -f /terasort")
```

Comment:
Can you prefix the `hdfs` commands with `$HADOOP_HOME/bin/hdfs`? On my machine I needed the prefix to run.

Reply:
Fixed.